source/PhysicalQuantity/Temperature.php

```diff
@@ -53,7 +55,9 @@ function ($x) {
             }
         );
         $newUnit->addAlias('F');
+        $newUnit->addAlias('f');
```

Comment:
What's the value of these alternate-case aliases?  Maybe it would be better in general to make unit comparison case-insensitive?